src/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import App from '@app/App';
+import MainApp from '@app/MainApp';
```

Comment:
@manaswinidas  ahmm what's the purpose of changing `App` to `MainApp` 🤔  anything significant?

Reply:
Not sure if this is the right way to handle it, as demonstrated earlier, I was getting `Uncaught Invariant Violation: Could not find "client" in the context or passed in as an option. Wrap the root component in an <ApolloProvider>, or pass an ApolloClient instance in via options.` when I introduced another hook for the useTokenQuery, I did my research and there were two ways to avoid it, one is bringing the ApolloClient and client out of App and the other is getting the query inside one of the child components. Since the verify token thing cannot be done inside the child components, I chose the latter. Not sure if this is the right way to handle this, would love the right way to handle this!